src/main/java/kobe/KobeN.java

```diff
@@ -0,0 +1,58 @@
+package kobe;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+public class KobeN {
+    private Storage storage;
+    private TaskList tasks; //ArrayList<Tasks> tasks
```

Comment:
attribute name should be "taskList" to demonstrate correctly what it means for. Tasks are separate tasks that maybe not related to each other, while taskList is different.